NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity.swift

```diff
@@ -25,7 +25,85 @@ import UIKit
 #elseif canImport(AppKit)
 import AppKit
 #endif
+// Mapping from XML/HTML character entity reference to character
+// From http://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references
+private let characterEntities: [ Substring: Character ] = [
+    // XML predefined entities:
+    "&quot;": "\"",
+    "&amp;": "&",
+    "&apos;": "'",
+    "&lt;": "<",
+    "&gt;": ">",
+    // HTML character entity references:
+    "&nbsp;": "\u{00a0}",
+    // ...
+    "&diams;": "♦"
+]
 
+extension String {
```

Comment:
Ideally, we don't want random extensions floating around (as this will confuse the other developers). So please consider the following options:
* Remove this extension and make `stringByDecodingHTMLEntitiesAnime` a member function of the `AnimeUnity` class;
* Move this extension to the file that it's being used and make it `fileprivate`;
* Move this extension to the `Utilities` group.